app/controllers/maintain/SetUpAfterSettlorDiedController.scala

```diff
@@ -60,12 +61,23 @@ class SetUpAfterSettlorDiedController @Inject()(
         (formWithErrors: Form[_]) =>
           Future.successful(BadRequest(view(formWithErrors))),
 
-        value => {
+        hasSettlorDied => {
           for {
-            updatedAnswers <- Future.fromTry(request.userAnswers.set(SetUpAfterSettlorDiedPage, value))
-            _              <- repository.set(updatedAnswers)
-          } yield Redirect(navigator.nextPage(SetUpAfterSettlorDiedPage, updatedAnswers))
+            answersWithSetUpValue <- Future.fromTry(request.userAnswers.set(SetUpAfterSettlorDiedPage, hasSettlorDied))
+            updatedAnswers <- addDefaultTypeOfTrust(hasSettlorDied, answersWithSetUpValue)
+            _ <- repository.set(updatedAnswers)
+          } yield {
+            Redirect(navigator.nextPage(SetUpAfterSettlorDiedPage, updatedAnswers))
+          }
         }
       )
   }
+
+  def addDefaultTypeOfTrust(hasSettlorDied: Boolean, userAnswers: UserAnswers): Future[UserAnswers] = {
+    if (hasSettlorDied) {
+      Future.fromTry(userAnswers.set(TypeOfTrustPage, TypeOfTrust.WillTrustOrIntestacyTrust))
+    } else {
+      Future.successful(userAnswers)
+    }
+  }
```

Comment:
I don't think we need this. The mapper at present doesn't expect this to be set for this scenario, and also it will mean things are rendered on the check-your-answers page that they didn't answer